app/views/candidate_interface/references/candidate_name/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.references_candidate_name'), @reference_candidate_name_form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_references_review_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @reference_candidate_name_form,
+      url: candidate_interface_references_update_candidate_name_path(@reference.id, return_to: params[:return_to]),
```

Comment:
I don't think we need this view as you are only able to edit this via the personal details section. This endpoint isn't in the routes either.

Reply:
My bad I think I added this in error thinking it was the 'name' attribute 🤦